src/producer.ts

```diff
@@ -94,6 +105,7 @@ export class Producer {
             message: {
               content: message,
               messageProperties: opts.messageProperties,
+              messageHeader: opts.messageHeader,
```

Comment:
Could the header be sent? are you sure?

Reply:
Sorry my bad, I added this before I knew it shouldn't be sent. I will fix this